package.json

```
@@ -15,7 +15,8 @@
     "node": ">=20.11.0"
   },
   "scripts": {
-    "build": "tsc -p .",
+    "build": "esbuild {.,pkg/**}/*.ts --platform=node --outdir=build --sourcemap",
+    "check": "tsc -p .",
```

型チェックだけなら`--noEmit`をつけるべきかと.
```suggestion
    "check": "tsc --noEmit -p .",
```

---

fdeafe8dbca9deba07918d6d33f835fcd917d824 で対応しました.